cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java

```diff
@@ -0,0 +1,250 @@
+package com.yugabyte.cdcsdk.testing;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.net.InetAddress;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.Duration;
+import java.util.Arrays;
+
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+import io.debezium.testing.testcontainers.ConnectorConfiguration;
+import io.debezium.testing.testcontainers.DebeziumContainer;
+
+/**
+ * Release test that verifies reading of multiple combination of operations from 
+ * a YugabyteDB database and writing to Kafka and then further to a PostgreSQL sink database 
+ *
+ * @author Vaibhav Kushwaha (vkushwaha@yugabyte.com)
+ */
+public class MultiOpsPostgresSinkConsumerIT {
```

Comment:
More test cases if not covered already
1. Single shard/multi shard tx
2. Multi tables
3. Multiple tablets
4. Multiple tasks
5. Restart/bootstrap
6. Composite key
7. data types (ALL have we covered? e.g multi dimensional arrays and all)
8. Unsupported data types

Reply:
I have modified the existing test for multiple tablets and added a new one to test bootstrap i.e. `YBBootstrapPostgresSinkConsumerIT`. For multi table setup or tests involving composite keys, some infra changes will be required - I have started working on the refactoring stuff and these remaining tests will be added a part of the next PR only.

The datatype tests are covered in the tests for the `debezium-connector-yugabytedb` repo.

Reply:
I hope above test cases are tracked for later.